examples/migrate_forseti/tutorial.md

```diff
@@ -0,0 +1,336 @@
+to match the region where the Forseti Server VM is deployed.
+
+On line 64, update the <walkthrough-editor-select-line
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  startLine=64
+  endLine=64
+  startCharacterOffset=21
+  endCharacterOffset=33>client_region</walkthrough-editor-select-line>
+to match the region where the Forseti Client VM is deployed.
+
+## Add Input Variables for Custom Configurations
+Starting with Forseti Security 2.23, Terraform will manage your server configuration file for you.  Configuration options will now be input variables that are defined in the Terraform module.  This will ensure upgrading Forseti will be as easy as possible going forward.
+
+Please identitify any Forseti server configuration variables that have been customized and add them to your <walkthrough-editor-open-file filePath="terraform-google-forseti/examples/migrate_forseti/main.tf">main.tf</walkthrough-editor-open-file>.
+
+For example, if you modified the e-mail connector recipient in the **forseti_conf_server.yaml** file in the Forseti Server GCS bucket:
+```yaml
+email_connector:
+      name: sendgrid
+      auth:
+        api_key: ...
+      sender: ...
+      recipient: myemail@example.com
+      data_format: csv
+```
+You will want to add the
+`forseti_email_recipient = myemail@example.com`
+to your <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="Add any Forseti Server Configuration Variables Here">main.tf</walkthrough-editor-select-regex>
+
+## Obtain and Run the Import Script
+This [import script](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/import.sh) will import the Forseti GCP resources into a local state file.
+
+```sh
+curl --location --remote-name https://raw.githubusercontent.com/forseti-security/terraform-google-forseti/master/helpers/import.sh
+chmod +x import.sh
+./import.sh -h
+```
+## Importing Existing Resources
+
+Initialize the Terraform state:
+
+```sh
+terraform init
+```
+
+Import the existing resources to the Terraform state, replacing the
+uppercase values with the aforementioned values:
+
+```sh
+./import.sh -m MODULE_LOCAL_NAME -o ORG_ID -p PROJECT_ID -s RESOURCE_NAME_SUFFIX -z GCE_ZONE [-n NETWORK_PROJECT_ID]
+```
+
+Observe the expected Terraform changes.  As stated in the introduction, if you have any
+questions about this process, please contact us by
+e-mail at discuss@forsetisecurity.org or on
+[Slack](https://forsetisecurity.slack.com/join/shared_invite/enQtNDIyMzg4Nzg1NjcxLTM1NTUzZmM2ODVmNzE5MWEwYzAwNjUxMjVkZjhmYWZiOGZjMjY3ZjllNDlkYjk1OGU4MTVhZGM4NzgyZjZhNTE).
+
+```sh
+terraform plan
+```
+
+### Terraform Changes
+
+Because there is not an exact mapping between the deprecated Python
+Installer and the Terraform module, some changes will occur when
+Terraform assumes management of the Forseti deployment.
+
+You should carefully review this section as well as the output from
```

Comment:
How is the review of `change` different from `plan`?  Also, we should call out the review of `plan` up at the step above.

Reply:
I've clarified and made it it's own page in the tutorial